nuxeo-features/nuxeo-automation/nuxeo-automation-test/src/test/java/org/nuxeo/ecm/automation/server/test/TestBulkDownloadOperation.java

```diff
@@ -81,8 +82,21 @@
 
     @Test
     public void iCanBulkDownloadFiles() throws IOException, OperationException, InterruptedException {
+        testWithNbFiles(MAX_DOC);
+    }
+
+    /**
+     * @since 10.1
+     */
+    @Test
+    public void iCanBulkDownloadASingleFile() throws IOException, OperationException, InterruptedException {
+        System.setProperty(RandomBug.MODE_PROPERTY, RandomBug.Mode.STRICT.toString());
```

Comment:
I don't think you need that anymore (or the import), do you?

Reply:
indeed, a leftover while trying to reproduce a random